Linker Error wb_robot_wwi_receive_text not in Controller.lib

**Describe the Bug**
I'm trying to build a controller for WeBots using Visual Studio 2019 Enterprise. When building the solution I get Linker Error "Robot.obj : error LNK2001: unresolved external symbol wb_robot_wwi_receive_text". All other external symbols link correctly. When opening Controller.lib in Notepad I can find all other external symbols, but not "wb_robot_wwi_receive_text".

**Steps to Reproduce**
1. Create a new controller using the Webots File > New > New Robot Controller
2. When prompted select "Microsoft Visual Studio"
3. Follow the instructions for linking, include libs, etc on https://www.cyberbotics.com/doc/guide/using-visual-studio-with-webots?version=cyberbotics:R2019a (More or less all of it is set correctly already)
4. Try to build the solution.

**Expected behavior**
No linker errors should occur

**System**
 - Operating System: Windows 10
 - Visual Studio 2019 Enterprise

--- a/src/controller/c/Controller.def
+++ b/src/controller/c/Controller.def
@@ -436,6 +436,7 @@ wb_robot_wait_for_user_input_event
 wb_robot_window_custom_function
 wb_robot_wwi_send
 wb_robot_wwi_receive
+wb_robot_wwi_receive_text
 wb_skin_get_bone_count
 wb_skin_get_bone_name
 wb_skin_get_bone_orientation

